recipes/optimum/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "optimum" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/optimum-{{ version }}.tar.gz
+  sha256: b51c54c47d7ff0f800cc7dfb833074292e4f3637df8494fe97c48cc382094c7d
+  # Remove the patch (adapted_pr_96.patch) 
+  # once version >1.0.0 is released.  
+  {% if version == "1.0.0" %}
+  patches:    
+    - adapted_pr_96.patch
+  {% endif %}
```

Comment:
Just one last thing but it's better to remove this jinja part as well and just change it manually.

I know that these are really helpful and make your job easier but outside of variables the bot just gets tripped up a lot on these when re-rendering and formatting the recipe so please be as minimal as possible with them. Conda-forge is just not really set up for too much jinja logic in the recipes. 😄 

Otherwise this seems good to go!

Reply:
Thank you. I will get rid of the jinja conditional. :smile:

Reply:
Done.